src/timeMachine/components/dynamicFluxFunctionsToolbar/FluxFunctionsToolbar.tsx

```diff
@@ -97,7 +97,10 @@ const DynamicFluxFunctionsToolbar: FC<Props> = (props: Props) => {
         spinnerComponent={<TechnoSpinner />}
       >
         <ErrorBoundary>
-          <FluxToolbarSearch onSearch={handleSearch} resourceName="Functions" />
+          <FluxToolbarSearch
+            onSearch={handleSearch}
+            resourceName="by Package or Function"
```

Comment:
this looks odd but the `FluxToolbarSearch` component is being used by several others. It takes `resourceName` prop and dynamically updates the placeholder depending on where it's used.

Reply:
a great example why naming things like props is important!